service-api/module/Application/src/Model/Service/ProcessingStatus/Service.php

```diff
@@ -101,12 +101,12 @@ public function getStatuses($ids)
             ],
             'fulfilled' => function ($response, $id) use (&$results) {
                 // Each successful response
-                $this->getLogger()->debug('We have a result for:' . $id);
+                //$this->getLogger()->debug('We have a result for:' . $id);
```

Comment:
not sure if this was right, but debug logging commented out here and below.

Reply:
good spot, that was to reduce noise while I was working on this, I'll reinstate it